ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -915,7 +915,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${PROJECT_DIR}/Crashlytics/Fabric.framework/run\" FABRIC_API_KEY_HERE FABRIC_SECRET_HERE";
+			shellScript = "if [ \"$releaseConfig\" = \"Release\" ]; then\n    \"${PROJECT_DIR}/Crashlytics/Fabric.framework/run\" FABRIC_API_KEY_HERE FABRIC_SECRET_HERE\nfi";
```

Comment:
Honestly, no idea.
I've read through a lot of Crashlytics' documentation and found only this as an example.